README.md

```diff
@@ -22,15 +22,14 @@ Latest image: `gke.gcr.io/gcp-filestore-csi-driver:v0.2.0`
 Also see [known issues](KNOWN_ISSUES.md) and [CHANGELOG](CHANGELOG.md).
 
 ### CSI Compatibility
-This plugin is compatible with CSI version 0.2.0 and 0.3.0.
+This plugin is compatible with CSI version 1.3.0.
 
 ### Kubernetes Compatibility
 
-| Filestore CSI Driver\Kubernetes Version | 1.10.5-1.11 | 1.12 | 1.13 | 1.14 |
-| --------------------------------------- | ----------- | ---- | ---- | ---- |
-| v0.1.0 (alpha)                          | yes         | no   | no   | no   |
-| v0.2.0 (alpha)                          | no          | yes  | yes  | yes  |
-| dev                                     | no          | yes  | yes  | yes  |
+| Filestore CSI Driver\Kubernetes Version | 1.10.5-1.11 | 1.12 | 1.13 | 1.14 | 1.15 | 1.16+ |
```

Comment:
we can remove the 1.10.5-1.11 column and just start from 1.12 for now

Reply:
Done